Yafc/Utils/CommandLineParser.cs

```diff
@@ -15,6 +16,15 @@ public static class CommandLineParser {
                 return options;
             }
 
+            if (args.Length == 1 && !args[0].StartsWith("--")) {
+                string path = Path.GetFullPath(args[0]);
+                ProjectDefinition? foundProject = Preferences.Instance.recentProjects.FirstOrDefault(
+                        x => x.path != null &&
+                             string.Equals(path, Path.GetFullPath(x.path), StringComparison.InvariantCultureIgnoreCase)
+                    );
+                return foundProject;
```

Comment:
This is a nice way to work with files we've opened before, but what about a fallback for new files? Possibly fall back to the most recent used settings or something? Might want to set path but not skip the welcome screen.

Reply:
Here's how I see the situation:

The goal of this feature is to skip the Welcome Screen when a Yafc project is opened with Windows file association. In other words, when a project is opened with something like a double-click on the Yafc project-file.

This functionality is not intended for a seamless creation of a new project. If a player uses this functionality to create a new project, we can just default to the Welcome Screen where they will create it as usual. 

Making this feature work for the creation of a new project may bring more bugs due to edge cases.

Reply:
In my case it was a copied or moved file. But yes, my preferabled solution would too be to open the welcome screen, optionally prefilled with something sane like the most recent paths.

Reply:
Hmmm, now that I think about it, you're right. The frustration of setting up the full path is much bigger than the frustration of crashing during the load if one had wrong pre-fills. I'll note it in the new PR https://github.com/shpaass/yafc-ce/pull/244. Let's continue our discussion there.